code/game/objects/items/cards_ids.dm

```diff
@@ -95,7 +95,33 @@
 	var/atom/A = target
 	if(!proximity && prox_check)
 		return
-	A.emag_act(user)
+	//Citadel changes: modular code misfiring, so we're bypassing into main code.
+	if(!uses)
+		user.visible_message("<span class='warning'>[src] emits a weak spark. It's burnt out!</span>")
+		playsound(src, 'sound/effects/light_flicker.ogg', 100, 1)
+		return
+	else if(uses <= 3)
+		playsound(src, 'sound/effects/light_flicker.ogg', 30, 1)	//Tiiiiiiny warning sound to let ya know your emag's almost dead
+
+	if(isturf(A))
+		return
+	if(istype(A,/obj/item/storage/lockbox))
+		A.emag_act(user)
+		uses = max(uses - 1, 0)
+		if(!uses)
+			user.visible_message("<span class='warning'>[src] fizzles and sparks. It seems like it's out of charges.</span>")
+			playsound(src, 'sound/effects/light_flicker.ogg', 100, 1)
+	if(istype(A,/obj/item/storage))
+		return
+	if(!(isobj(A) || issilicon(A) || isbot(A) || isdrone(A)))
+		return
+	else

```

Comment:
This `else` here isn't necessary, as the check right above is an `if(); return`, and all of the other checks have a return at the end of them